 Network rinha-klimber-2024q1  Creating
 Network rinha-klimber-2024q1  Created
 Container klimber-db-1  Creating
 Container klimber-db-1  Created
 Container klimber-backend01-1  Creating
 Container klimber-backend02-1  Creating
 Container klimber-backend02-1  Created
 Container klimber-backend01-1  Created
 Container klimber-nginx-1  Creating
 Container klimber-nginx-1  Created
Attaching to backend01-1, backend02-1, db-1, nginx-1
db-1         | The files belonging to this database system will be owned by user "postgres".
db-1         | This user must also own the server process.
db-1         | 
db-1         | The database cluster will be initialized with locale "en_US.utf8".
db-1         | The default database encoding has accordingly been set to "UTF8".
db-1         | The default text search configuration will be set to "english".
db-1         | 
db-1         | Data page checksums are disabled.
db-1         | 
db-1         | fixing permissions on existing directory /var/lib/postgresql/data ... ok
db-1         | creating subdirectories ... ok
db-1         | selecting dynamic shared memory implementation ... posix
db-1         | selecting default max_connections ... 100
db-1         | selecting default shared_buffers ... 128MB
db-1         | selecting default time zone ... Etc/UTC
db-1         | creating configuration files ... ok
db-1         | running bootstrap script ... ok
nginx-1      | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx-1      | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx-1      | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx-1      | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
nginx-1      | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx-1      | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx-1      | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx-1      | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx-1      | /docker-entrypoint.sh: Configuration complete; ready for start up
db-1         | performing post-bootstrap initialization ... ok
db-1         | initdb: warning: enabling "trust" authentication for local connections
db-1         | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
db-1         | syncing data to disk ... ok
db-1         | 
db-1         | 
db-1         | Success. You can now start the database server using:
db-1         | 
db-1         |     pg_ctl -D /var/lib/postgresql/data -l logfile start
db-1         | 
db-1         | waiting for server to start....2024-02-18 15:18:01.777 UTC [49] LOG:  starting PostgreSQL 15.6 (Debian 15.6-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1         | 2024-02-18 15:18:01.780 UTC [49] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1         | 2024-02-18 15:18:01.790 UTC [52] LOG:  database system was shut down at 2024-02-18 15:18:01 UTC
db-1         | 2024-02-18 15:18:01.835 UTC [49] LOG:  database system is ready to accept connections
db-1         |  done
db-1         | server started
db-1         | CREATE DATABASE
db-1         | 
db-1         | 
db-1         | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/01_init.sql
db-1         | CREATE TABLE
db-1         | CREATE TABLE
db-1         | CREATE TABLE
db-1         | ALTER TABLE
db-1         | ALTER TABLE
db-1         | 
db-1         | 
db-1         | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/02_prefill.sql
db-1         | INSERT 0 5
db-1         | INSERT 0 5
db-1         | 
db-1         | 
db-1         | 2024-02-18 15:18:02.242 UTC [49] LOG:  received fast shutdown request
db-1         | waiting for server to shut down....2024-02-18 15:18:02.244 UTC [49] LOG:  aborting any active transactions
db-1         | 2024-02-18 15:18:02.252 UTC [49] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
db-1         | 2024-02-18 15:18:02.252 UTC [50] LOG:  shutting down
db-1         | 2024-02-18 15:18:02.254 UTC [50] LOG:  checkpoint starting: shutdown immediate
db-1         | 2024-02-18 15:18:02.739 UTC [50] LOG:  checkpoint complete: wrote 937 buffers (5.7%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.369 s, sync=0.102 s, total=0.488 s; sync files=310, longest=0.075 s, average=0.001 s; distance=4274 kB, estimate=4274 kB
db-1         | 2024-02-18 15:18:02.744 UTC [49] LOG:  database system is shut down
db-1         |  done
db-1         | server stopped
db-1         | 
db-1         | PostgreSQL init process complete; ready for start up.
db-1         | 
db-1         | 2024-02-18 15:18:02.866 UTC [1] LOG:  starting PostgreSQL 15.6 (Debian 15.6-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1         | 2024-02-18 15:18:02.866 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db-1         | 2024-02-18 15:18:02.866 UTC [1] LOG:  listening on IPv6 address "::", port 5432
db-1         | 2024-02-18 15:18:02.871 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1         | 2024-02-18 15:18:02.878 UTC [69] LOG:  database system was shut down at 2024-02-18 15:18:02 UTC
db-1         | 2024-02-18 15:18:02.885 UTC [1] LOG:  database system is ready to accept connections
nginx-1      | 2024/02/18 15:18:03 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.18.0.3:8080/clientes/1/extrato", host: "localhost:9999"
nginx-1      | 2024/02/18 15:18:03 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.18.0.4:8081/clientes/1/extrato", host: "localhost:9999"
backend02-1  | 
backend02-1  |   .   ____          _            __ _ _
backend02-1  |  /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
backend02-1  | ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
backend02-1  |  \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
backend02-1  |   '  |____| .__|_| |_|_| |_\__, | / / / /
backend02-1  |  =========|_|==============|___/=/_/_/_/
backend02-1  |  :: Spring Boot ::                (v3.2.2)
backend02-1  | 
backend01-1  | 
backend01-1  |   .   ____          _            __ _ _
backend01-1  |  /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
backend01-1  | ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
backend01-1  |  \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
backend01-1  |   '  |____| .__|_| |_|_| |_\__, | / / / /
backend01-1  |  =========|_|==============|___/=/_/_/_/
backend01-1  |  :: Spring Boot ::                (v3.2.2)
backend01-1  | 
nginx-1      | 2024/02/18 15:18:05 [error] 29#29: *4 no live upstreams while connecting to upstream, client: 172.18.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://backend/clientes/1/extrato", host: "localhost:9999"
backend02-1  | 2024-02-18T15:18:05.423Z  INFO 1 --- [           main] c.k.s.SpringRinha2024Q1Application       : Starting SpringRinha2024Q1Application v0.0.1-SNAPSHOT using Java 21.0.2 with PID 1 (/spring-rinha-2024-q1.jar started by root in /)
backend02-1  | 2024-02-18T15:18:05.432Z  INFO 1 --- [           main] c.k.s.SpringRinha2024Q1Application       : No active profile set, falling back to 1 default profile: "default"
backend01-1  | 2024-02-18T15:18:05.632Z  INFO 1 --- [           main] c.k.s.SpringRinha2024Q1Application       : Starting SpringRinha2024Q1Application v0.0.1-SNAPSHOT using Java 21.0.2 with PID 1 (/spring-rinha-2024-q1.jar started by root in /)
backend01-1  | 2024-02-18T15:18:05.634Z  INFO 1 --- [           main] c.k.s.SpringRinha2024Q1Application       : No active profile set, falling back to 1 default profile: "default"
nginx-1      | 2024/02/18 15:18:07 [error] 29#29: *5 no live upstreams while connecting to upstream, client: 172.18.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://backend/clientes/1/extrato", host: "localhost:9999"
backend02-1  | 2024-02-18T15:18:09.428Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JDBC repositories in DEFAULT mode.
nginx-1      | 2024/02/18 15:18:09 [error] 29#29: *6 no live upstreams while connecting to upstream, client: 172.18.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://backend/clientes/1/extrato", host: "localhost:9999"
backend02-1  | 2024-02-18T15:18:09.727Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 209 ms. Found 3 JDBC repository interfaces.
backend01-1  | 2024-02-18T15:18:09.816Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JDBC repositories in DEFAULT mode.
backend01-1  | 2024-02-18T15:18:10.216Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 393 ms. Found 3 JDBC repository interfaces.
nginx-1      | 2024/02/18 15:18:11 [error] 29#29: *7 no live upstreams while connecting to upstream, client: 172.18.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://backend/clientes/1/extrato", host: "localhost:9999"
backend02-1  | 2024-02-18T15:18:12.733Z  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port 8081 (http)
backend02-1  | 2024-02-18T15:18:12.831Z  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
backend02-1  | 2024-02-18T15:18:12.832Z  INFO 1 --- [           main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.18]
backend01-1  | 2024-02-18T15:18:13.234Z  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port 8080 (http)
backend01-1  | 2024-02-18T15:18:13.332Z  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
backend01-1  | 2024-02-18T15:18:13.333Z  INFO 1 --- [           main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.18]
backend02-1  | 2024-02-18T15:18:13.431Z  INFO 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
backend02-1  | 2024-02-18T15:18:13.433Z  INFO 1 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 7708 ms
nginx-1      | 2024/02/18 15:18:13 [error] 29#29: *8 no live upstreams while connecting to upstream, client: 172.18.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://backend/clientes/1/extrato", host: "localhost:9999"
backend01-1  | 2024-02-18T15:18:14.021Z  INFO 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
backend01-1  | 2024-02-18T15:18:14.023Z  INFO 1 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 8006 ms
backend02-1  | 2024-02-18T15:18:15.026Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
nginx-1      | 2024/02/18 15:18:15 [error] 29#29: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.18.0.4:8081/clientes/1/extrato", host: "localhost:9999"
nginx-1      | 2024/02/18 15:18:15 [error] 29#29: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.18.0.3:8080/clientes/1/extrato", host: "localhost:9999"
backend01-1  | 2024-02-18T15:18:15.741Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
backend02-1  | 2024-02-18T15:18:16.028Z  INFO 1 --- [           main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@575cabf0
backend02-1  | 2024-02-18T15:18:16.030Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
backend01-1  | 2024-02-18T15:18:16.647Z  INFO 1 --- [           main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@433d9680
backend01-1  | 2024-02-18T15:18:16.648Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
nginx-1      | 2024/02/18 15:18:17 [error] 29#29: *12 no live upstreams while connecting to upstream, client: 172.18.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://backend/clientes/1/extrato", host: "localhost:9999"
nginx-1      | 2024/02/18 15:18:19 [error] 29#29: *13 no live upstreams while connecting to upstream, client: 172.18.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://backend/clientes/1/extrato", host: "localhost:9999"
backend02-1  | 2024-02-18T15:18:20.824Z  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port 8081 (http) with context path ''
backend02-1  | 2024-02-18T15:18:20.919Z  INFO 1 --- [           main] c.k.s.SpringRinha2024Q1Application       : Started SpringRinha2024Q1Application in 18.086 seconds (process running for 20.954)
backend01-1  | 2024-02-18T15:18:21.421Z  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port 8080 (http) with context path ''
backend01-1  | 2024-02-18T15:18:21.429Z  INFO 1 --- [           main] c.k.s.SpringRinha2024Q1Application       : Started SpringRinha2024Q1Application in 18.411 seconds (process running for 21.458)
nginx-1      | 2024/02/18 15:18:21 [error] 29#29: *14 no live upstreams while connecting to upstream, client: 172.18.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://backend/clientes/1/extrato", host: "localhost:9999"
nginx-1      | 2024/02/18 15:18:23 [error] 29#29: *15 no live upstreams while connecting to upstream, client: 172.18.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://backend/clientes/1/extrato", host: "localhost:9999"
nginx-1      | 2024/02/18 15:18:25 [error] 29#29: *16 no live upstreams while connecting to upstream, client: 172.18.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://backend/clientes/1/extrato", host: "localhost:9999"
backend02-1  | 2024-02-18T15:18:27.625Z  INFO 1 --- [nio-8081-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
backend02-1  | 2024-02-18T15:18:27.625Z  INFO 1 --- [nio-8081-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
backend02-1  | 2024-02-18T15:18:27.626Z  INFO 1 --- [nio-8081-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 1 ms
backend01-1  | 2024-02-18T15:18:33.826Z  INFO 1 --- [nio-8080-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
backend01-1  | 2024-02-18T15:18:33.827Z  INFO 1 --- [nio-8080-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
backend01-1  | 2024-02-18T15:18:33.829Z  INFO 1 --- [nio-8080-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 2 ms
nginx-1 exited with code 0
nginx-1 exited with code 0
backend01-1  | 2024-02-18T15:22:42.684Z  INFO 1 --- [ionShutdownHook] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
backend01-1  | 2024-02-18T15:22:42.690Z  INFO 1 --- [ionShutdownHook] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
backend02-1  | 2024-02-18T15:22:42.690Z  INFO 1 --- [ionShutdownHook] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
backend02-1  | 2024-02-18T15:22:42.715Z  INFO 1 --- [ionShutdownHook] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
backend01-1 exited with code 0
backend01-1 exited with code 143
backend02-1 exited with code 0
backend02-1 exited with code 143
db-1         | 2024-02-18 15:22:43.721 UTC [1] LOG:  received fast shutdown request
db-1         | 2024-02-18 15:22:43.725 UTC [1] LOG:  aborting any active transactions
db-1         | 2024-02-18 15:22:43.727 UTC [1] LOG:  background worker "logical replication launcher" (PID 72) exited with exit code 1
db-1         | 2024-02-18 15:22:43.728 UTC [67] LOG:  shutting down
db-1         | 2024-02-18 15:22:43.734 UTC [67] LOG:  checkpoint starting: shutdown immediate
db-1         | 2024-02-18 15:22:43.906 UTC [67] LOG:  checkpoint complete: wrote 677 buffers (4.1%); 0 WAL file(s) added, 0 removed, 1 recycled; write=0.014 s, sync=0.137 s, total=0.179 s; sync files=30, longest=0.081 s, average=0.005 s; distance=16418 kB, estimate=16418 kB
db-1         | 2024-02-18 15:22:43.910 UTC [1] LOG:  database system is shut down
db-1 exited with code 0
